app/Http/Controllers/PayloadHistoryController.php

```diff
@@ -93,4 +93,61 @@ public function destroy(Request $request, PayloadHistory $history)
             ]);
         }
     }
+
+    public function recheck($id) {
+        $errorFields = [];
+        $result = [];
+        $payloadHistory = PayloadHistory::findOrFail($id);
+        $params = json_decode(json_encode(json_decode($payloadHistory->params)), true);
+        $payloads = $payloadHistory->webhook->payloads;
+
+        for ($i = 0; $i < count($payloads); $i++) {
+            $content = $payloadHistory->webhook->payloads[$i]->content;
+            $value = $this->getStringsBetweebBrackets($content);
+            $result = array_merge($result, $value);
+        }
+        for ($i = 0; $i < count($result); $i++) {
+            try {
+                $this->getValues($params, $result[$i]);
+            } catch (\Throwable $th) {
+                array_push($errorFields, trim($result[$i]));
+                continue;
+            }
+        }
+        if(count($errorFields) != 0) {
+            return response()->json([
+                'error' => true,
+                'data' => array_unique($errorFields)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadHistoryController.php</br>**Line**: 120</br>Each line in an array declaration must end in a comma